hack/lib/install.sh

```diff
@@ -96,12 +96,74 @@ verify_docker_installed(){
 
 # install CNI plugins
 function install_cni_plugins() {
+  CNI_DOWNLOAD_ADDR=${1:-"https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz"}
+  CNI_PKG=${CNI_DOWNLOAD_ADDR##*/}
+  OVERWRITE=${2:-"false"}
+
   # install CNI plugins if not exist
   if [ ! -f "/opt/cni/bin/loopback" ]; then
-    echo -e "install CNI plugins..."
-    mkdir -p /opt/cni/bin
-    wget https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz
-    tar Cxzvf /opt/cni/bin cni-plugins-linux-amd64-v1.1.1.tgz
+    echo -e "start installing CNI plugins..."
+    sudo mkdir -p /opt/cni/bin
+    wget ${CNI_DOWNLOAD_ADDR}
+    if [ ! -f ${CNI_PKG} ]; then
+      echo -e "cni plugins package does not exits"
+      exit 1
+    fi
+    sudo tar Cxzvf /opt/cni/bin ${CNI_PKG}
+    rm -rf ${CNI_PKG}
+    if [ ! -f "/opt/cni/bin/loopback" ]; then
+      echo -e "the ${CNI_PKG} package does not contain a loopback file."
+      exit 1
+    fi
+
+    # create CNI netconf file
+    CNI_CONFIG="/etc/cni/net.d/10-containerd-net.conflist"
+    if [ -f ${CNI_CONFIG} ]; then
+      if [ ${CNI_CONF_OVERWRITE} != "true" ]; then
+        echo -e "CNI netconf file already exist and no need to create"
+        return
+      fi
+      echo -e "existing file /etc/cni/net.d/10-containerd-net.conflist overwrites installation"
```

Comment:
```suggestion
      echo -e "Configuring cni, if /etc/cni/net.d/10-containerd-net.conflist exist, will be overwrrite..."
```
If it‘s first time to install, it still not exist now :)